druid/src/widget/label.rs

```diff
@@ -80,9 +84,21 @@ impl<T: Data> Label<T> {
         self
     }
 
+    /// Set text color.
```

Comment:
I would indicate that these are 'builder-style' methods, and I would also consider having mutating versions of these, called `set_text_color` and `set_text_size`, for when we want to modify things directly at runtime? Although it's possible we want that to only be done through the environment; I'm not totally sure what patterns we'll want?

My *hunch* is that it makes sense to allow a parent to modify these on a child directly when needed, but I'm not 100% confident..

Reply:
I went with adding the setters. I figure if it causes problems or we never use it in practice we can remove?